DocumentDataSubmission/src/main/resources/wsdl/AdapterComponentDDS.wsdl

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- This wsdl file is for an XDS.b Document Repository Actor -->
+<!-- This wsdl file is for an XDS.b Document Registry Actor -->
 <definitions xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap12/"
              xmlns:tns="urn:gov:hhs:fha:nhinc:adaptercomponentdds:data"
```

Comment:
Since we use xds, we should change our namespace to use xds instead of "dds".  Suggestion to use adaptercomponentxds:data for this file. Make sure to check on other files as well.